shell/platform/darwin/ios/framework/Source/SemanticsObject.mm

```diff
@@ -294,23 +294,22 @@ - (instancetype)initWithBridge:(fml::WeakPtr<flutter::AccessibilityBridgeIos>)br
     _bridge = bridge;
     _uid = uid;
     _children = [[NSMutableArray alloc] init];
-    _childrenInHitTestOrder = [[NSMutableArray alloc] init];
+    _childrenInHitTestOrder = [[NSArray alloc] init];
   }
 
   return self;
 }
 
 - (void)dealloc {
   for (SemanticsObject* child in _children) {
-    [child privateSetParent:nil];
+    child.parent = nil;
   }
   [_children removeAllObjects];
```

Comment:
Can we remove this too? Hopefully we don't have code that is relying on the contents of an array that came from a since-deallocated object changing out from under it...

Reply:
I'll remove it in the ARC migration.